x-pack/plugins/integration_assistant/server/config.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema, type TypeOf } from '@kbn/config-schema';
+import type { PluginConfigDescriptor } from '@kbn/core/server';
+
+export const configSchema = schema.object({
+  enabled: schema.boolean({ defaultValue: true }),
```

Comment:
Be careful, this should be `disabled` by default.
cc @patrykkopycinski

Reply:
Fixed in https://github.com/elastic/kibana/pull/184296/commits/a10dce03bb4cf5286e0e37dde993b784204fbcd1